addons/im_livechat/static/src/core/discuss_patch.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="im_livechat.Discuss" t-inherit="mail.Discuss" t-inherit-mode="extension">
+        <xpath expr="//div" position="before">
+            <ControlPanel/>
+        </xpath>
+        <xpath expr="//div" position="attributes">
+            <!-- Substract the height of the ControlPanel component -->
+            <attribute name="style">max-height: calc(100% - 58px)!important</attribute> 
```

Comment:
This looks very hacky and not robust, and I'm not even sure to understand the purpose.

Reply:
It is tricky because the component is scrollable and the overflow hidden. This should be much better already: https://github.com/odoo/odoo/pull/137623/files#diff-439a221c51f6b01171266dfcd7035e99a0b443e3296c1f16b3492fc4e92e0975R1